splunk-otel-android/src/main/java/io/opentelemetry/rum/internal/instrumentation/slowrendering/SlowRenderListener.java

```diff
@@ -194,7 +189,7 @@ private void reportSlowRenders() {
         try {
             activities.forEach((activity, listener) -> reportSlow(listener));
         } catch (Exception e) {
-            Log.w(LOG_TAG, "Exception while processing frame metrics", e);
+            Log.w("OpenTelemetryRum", "Exception while processing frame metrics", e);
```

Comment:
we should still make a constant field for that `OpenTelemetryRum` even if it's not `LOG_TAG`. What happened to that anyway?

Reply:
`LOG_TAG` is placed in `SplunkRum`, can't use it here.

> we should still make a constant field for that `OpenTelemetryRum`

👍